tests/settings/test_settings_typo_tolerance_meilisearch.py

```diff
@@ -0,0 +1,60 @@
+DEFAUT_TYPO_TOLERANCE =   {
+    'enabled': True,
+    'minWordSizeForTypos': {
+        'oneTypo': 5,
+        'twoTypos': 9,
+    },
+    'disableOnWords': [],
+    'disableOnAttributes': [],
+}
+
+NEW_TYPO_TOLERANCE =   {
+    'enabled': True,
+    'minWordSizeForTypos': {
+        'oneTypo': 6,
+        'twoTypos': 10,
+    },
+    'disableOnWords': [],
+    'disableOnAttributes': ['title'],
+}
+
+def test_get_typo_tolerance_default(empty_index):
+    """Tests getting default typo_tolerance."""
+    response = empty_index().get_typo_tolerance()
+    assert isinstance(response, dict)
+    assert response == DEFAUT_TYPO_TOLERANCE
+    print(response)
+
+def test_update_typo_tolerance(empty_index):
+    """Tests updating typo_tolerance."""
+    index = empty_index()
+    response = index.update_typo_tolerance(NEW_TYPO_TOLERANCE)
+    assert isinstance(response, dict)
+    assert 'uid' in response
+    update = index.wait_for_task(response['uid'])
+    assert update['status'] == 'succeeded'
+    response = index.get_typo_tolerance()
+    assert isinstance(response, dict)
+    for typo_tolerance in NEW_TYPO_TOLERANCE:
+        assert typo_tolerance in response
```

Comment:
This will only check the keys and not the values, is that what you mean to do? If so it's fine.